src/wasm/wasm-binary.cpp

```diff
@@ -2828,12 +2829,29 @@ BinaryConsts::ASTNodes WasmBinaryBuilder::readExpression(Expression*& curr) {
       break;
     case BinaryConsts::Else:
       curr = nullptr;
-      continueControlFlow(BinaryLocations::Else, startPos);
+      if (DWARF && currFunction) {
+        assert(!controlFlowStack.empty());
+        auto currControlFlow = controlFlowStack.back();
+        currFunction
+          ->delimiterLocations[currControlFlow][BinaryLocations::Else] =
+          startPos - codeSectionLocation;
+      }
       break;
     case BinaryConsts::Catch:
+    case BinaryConsts::CatchAll: {
```

Comment:
@aheejin actually this fails to compile, since the binary code for CatchAll is 5, the same as for Else, so it says we have a duplicate entry in the switch.

Is it correct that CatchAll has the same code as Else?

Reply:
Yes that's correct; we are doing that to reuse the opcode because `else` cannot occur with a `try`.